internal/util/importutilv2/parquet/field_reader.go

```diff
@@ -261,7 +261,7 @@ func ReadBinaryData(pcr *FieldReader, count int64) (any, error) {
 		case arrow.LIST:
 			listReader := chunk.(*array.List)
 			if !isVectorAligned(listReader.Offsets(), pcr.dim, dataType) {
-				return nil, merr.WrapErrImportFailed("%s not aligned", dataType.String())
+				return nil, merr.WrapErrImportFailed(fmt.Sprintf("%s's dimension not aligned", dataType.String()))
```

Comment:
it would be better to tell the user what is the expected value, and what is the actual value.